Kitodo/src/main/webapp/pages/inc_Prozessverwaltung/prozess_box_Properties.xhtml

```diff
@@ -46,148 +46,36 @@
-                        </h:commandLink>
-                    </td>
-
-                </tr>
-            </t:dataList>
-            -->
-
-        </t:dataList>
-
-
-        <!--
-
-        <t:dataList var="container" value="#{ProzessverwaltungForm.containers}" rowCountVar="rowCount" rowIndexVar="rowIndex">
-            <t:dataList var="proc" value="#{ProzessverwaltungForm.containerlessProperties}" rowCountVar="propCount" rowIndexVar="propInd">
-                <tr rendered="#{container == 0}" class="standardTable_Row1">
-                    <td>
-                        <h:outputText value="#{proc.name}" />
-                    </td>
-                    <td>
-                        <h:outputText value="#{proc.value}" />
+                        <h:outputText value="#{property.value}" rendered="#{property.type.name !=  'link'}"/>
+                        <h:outputLink title="#{property.value}" value="#{property.value}" rendered="#{property.type.name ==  'link'}"
+                                          target="_blank">
+                            <h:outputText value="#{property.value}"/>
+                        </h:outputLink>
                     </td>
                     <td class="standardTable_ColumnCentered">
                         <h:commandLink action="ProzessverwaltungBearbeiten" title="#{msgs.bearbeiten}">
                             <h:graphicImage value="/pages/images/buttons/edit.gif" alt="edit"/>
-                            <t:updateActionListener property="#{ProzessverwaltungForm.processProperty}" value="#{proc}" />
-                            <t:updateActionListener property="#{ProzessverwaltungForm.container}" value="0" />
-                            <t:updateActionListener property="#{ProzessverwaltungForm.modusBearbeiten}" value="eigenschaft" />
+                            <t:updateActionListener property="#{ProzessverwaltungForm.property}" value="#{property}"/>
+                            <t:updateActionListener property="#{ProzessverwaltungForm.modusBearbeiten}" value="eigenschaft"/>
                         </h:commandLink>
-                        <h:commandLink action="#{ProzessverwaltungForm.duplicateContainer}" title="#{msgs.duplicate}">
+                        <h:commandLink action="#{ProzessverwaltungForm.duplicateProperty}" title="#{msgs.duplicate}">
                             <h:graphicImage value="/pages/images/buttons/copy.gif" alt="copy"/>
-                            <t:updateActionListener property="#{ProzessverwaltungForm.processProperty}" value="#{proc}" />
-                            <t:updateActionListener property="#{ProzessverwaltungForm.container}" value="0" />
+                            <t:updateActionListener property="#{ProzessverwaltungForm.property}" value="#{property}"/>
                         </h:commandLink>
                     </td>
                 </tr>
             </t:dataList>
 
+            <!--
```

Comment:
If this is not needed any more, remove it.

Reply:
I think that one is going to be restored but I'm not 100% sure.

Reply:
Who know this?

Reply:
Hard to say, so I can remove this part also and in case it had some usage I will restore it later...